pymatgen/electronic_structure/plotter.py

```diff
@@ -2925,23 +2925,24 @@ def plot_points(points, lattice=None, coords_are_cartesian=False, fold=False, ax
 
 
 @add_fig_kwargs
-def plot_brillouin_zone_from_kpath(kpath, **kwargs):
+def plot_brillouin_zone_from_kpath(kpath, ax=None, **kwargs):
```

Comment:
Shouldn't you just be able to use `ax=ax` here and have it be propagated through the `**kwargs`?